book-example/src/format/config.md

```diff
@@ -71,14 +72,39 @@ The following preprocessors are available and included by default:
 - `index`: Convert all chapter files named `README.md` into `index.md`. That is
   to say, all `README.md` would be rendered to an index file `index.html` in the
   rendered book.
-
+- `emoji`: Convert `:emoji:` text into Emojis. eg: `:smile:` to :smile:
 
 **book.toml**
 ```toml
 [build]
 build-dir = "build"
 create-missing = false
-preprocess = ["links", "index"]
+
+[preprocess]
+links = true
+index = true
+emoji = true
+```
+
+#### Locking a Preprocessor dependency to a renderer
+
+You can explicitly specify that a preprocessor should run for a renderer by binding the two together.
+
+```
+[preprocessor.mathjax]
+renderers = ["html"]  # mathjax only makes sense with the HTML renderer
+```
+
+#### Nested Configuration
```

Comment:
This section will probably need to mention that, like renderers, each preprocessor will always need to be given its own table (e.g. `[preprocessor.mathjax]`). From there, passing extra configuration to the preprocessor is just a case of adding key-value pairs to the table, so there isn't really a separate "*style of configuration*".